consoles/sshXtermIPMI.pm

```diff
@@ -22,6 +22,7 @@ use testapi 'get_required_var';
 require IPC::System::Simple;
 use autodie ':all';
 use File::Which;
+use testapi 'get_var';
```

Comment:
Please add that in line 21. However IMHO using the test API from within the consoles is bad design

Reply:
I will think whether this can be avoided.